static/styles/left-sidebar.scss

```diff
@@ -345,6 +345,8 @@ ul.expanded_private_messages {
     margin-left: 0px;
     padding-bottom: 2px;
     margin-top: 3px;
+    max-height: 200px;
```

Comment:
How well does this do in responsive situations?  We might need to modify `left_userlist_get_new_heights` (which is the hacky thing that handles similar things elsewhere)...

Reply:
Good question. Let me check.